PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/hdinsight/FxClassLoader.java

```diff
@@ -48,7 +48,9 @@
 	
 	public static void loadJavaFxForJobView(Composite composite, String url) {
 		if(!isJavaFxLoaded) {
-			PluginUtil.forceInstallPluginUsingMarketPlaceAsync(efxclipsePluginSymbolicName, efxclipseMarketplaceURL);
+			if (PluginUtil.isJavaVersionHigherThanTarget(1.7) == false) {
```

Comment:
`if (!PluginUtil.isJavaVersionHigherThanTarget(1.7)) {` would be enough.

Reply:
Fixed

---
In reply to: [144220040](https://github.com/Microsoft/azure-tools-for-java/pull/1098#discussion_r144220040) [](ancestors = 144220040)